packages/editor/src/components/global-styles-provider/index.js

```diff
@@ -148,10 +172,13 @@ function useGlobalStylesBaseConfig() {
 	const baseConfig = useSelect( ( select ) => {
 		const { __experimentalGetCurrentThemeBaseGlobalStyles, canUser } =
 			select( coreStore );
-
+		const currentPost = select( editorStore ).getCurrentPost();
 		return (
-			canUser( 'read', { kind: 'root', name: 'theme' } ) &&
-			__experimentalGetCurrentThemeBaseGlobalStyles()
+			canUser( 'read', {
```

Comment:
We could probably remove this since all roles that can read a post in the editor will need to read access to the styles object.

Reply:
This makes sense to me 👍

Reply:
> What's faster than not having to perform the check at all 😁

That's the most performant "check" 😄